app/questionnaire/navigator.py

```diff
@@ -199,8 +191,12 @@ def get_blocks(self):
         blocks = []
 
         for group_index, group in enumerate(SchemaHelper.get_groups(self.survey_json)):
-            no_of_repeats = 1
+            if 'skip_condition' in group:
```

Comment:
Might make sense one day but let's do it in a feature driven manor then just add it now. Looks at the S3 bucket stuff, isn't used.